Cargo.toml

```diff
@@ -29,6 +29,7 @@ env_logger = "0.8.4"
 eventual = "0.1.7"
 flate2 = "1.0.18"
 fs2 = "0.4.3"
+futures = ""
```

Comment:
This is valid? We're going to have to pin it at a specific version anyway for crates.io publishing...